api-reference/10 UI Components/dxDataGrid/1 Configuration/summary/calculateCustomSummary.md

```diff
@@ -56,7 +56,6 @@ A summary value calculation is conducted in three stages: *start* - the **totalV
                                 break;
                             case "finalize":
                                 // Assigning the final value to "totalValue" here
-                                break;
```

Comment:
I vote for keeping the break. Sure, the code works without it, but it's good to show it just in case somebody wants to add a default case.